spec/features/finance/npq/course_payment_breakdown_spec.rb

```diff
@@ -281,4 +279,72 @@ def overall_vat
   def overall_total
     total_payment + overall_vat
   end
+
+  def statement_declarations_per_contract(contract)
```

Comment:
unfortunately these tests seem to be a based on a copy/pasted version of the actual implementations. i have left these as is for now but we may want to later convert these to assert against something a bit more concrete.